src/lib_ccx/params.c

```diff
@@ -309,10 +309,15 @@ void usage (void)
 	mprint ("                       (DEFAULT is -1)\n");
 	mprint ("                 -cc2: When in srt/sami mode, process captions in channel 2\n");
 	mprint ("                       instead of channel 1.\n");
-	mprint ("-svc --service N,N...: Enabled CEA-708 captions processing for the listed\n");
+	mprint ("-svc --service N1[cs1],N2[cs2]...:\n");
+	mprint ("                       Enable CEA-708 captions processing for the listed\n");
 	mprint ("                       services. The parameter is a command delimited list\n");
 	mprint ("                       of services numbers, such as \"1,2\" to process the\n");
 	mprint ("                       primary and secondary language services.\n");
+	mprint ("                       Pass \"all\" to process all services found.\n");
+	mprint ("                       To tell encoder what kind of charset was used, pass its name after\n");
+	mprint ("                       service number (e.g. \"1[EUC-KR],3\" or \"all[EUC-KR]\"\n");
+	mprint ("\n");
```

Comment:
Right now, by default it replaces 16-bit chars with "?" symbol. I am keeping this task in my mind and will fix while refactoring encoding part.